gradle/libs.versions.toml

```diff
@@ -0,0 +1,122 @@
+[versions]
+acraCore = "5.11.3"
+appcompat = "1.7.0"
+autoServiceKsp = "1.2.0"
+biometric = "1.2.0-alpha05"
+buildkonfigGradlePlugin = "0.15.1"
+colorpicker = "1.1.0"
+conscryptAndroid = "2.5.2"
+constraintlayout = "2.1.4"
+coreKtx = "1.13.1"
+desugar_jdk_libs_nio = "2.0.4"
+dokkaGradlePlugin = "1.9.10"
+espressoCore = "3.6.1"
+fuzzywuzzy = "1.4.0"
+glide = "4.16.0"
+glideTransformations = "4.3.0"
+gradle = "8.2.2"
+guava = "33.2.1-android"
+jacksonModuleKotlin = "2.13.1"
+json = "20240303"
+junit = "4.13.2"
+junitKtx = "1.2.1"
+junitVersion = "1.2.1"
+juniversalchardet = "2.5.0"
+kotlinGradlePlugin = "1.9.23"
+kotlinxCoroutinesCore = "1.8.0"
+ksp = "4.16.0"
+lifecycleLivedataKtx = "2.8.3"
+lifecycleViewmodelKtx = "2.8.3"
+material = "1.12.0"
+media3Cast = "1.4.0"
```

Comment:
Why is every media3 dep separate entries here? Is not the entire point of this system to union the dependencies, so you only have to define a media3 ver and all media3Cast,media3Common,media3DatasourceOkhttp ect gets the same?

Reply:
Yes, it's up to us if we want to union them together no issue we can do this